beacon_node/lighthouse_network/src/types/globals.rs

```diff
@@ -121,6 +120,16 @@ impl<E: EthSpec> NetworkGlobals<E> {
         self.listen_multiaddrs.read().clone()
     }
 
+    /// Returns true if this node is configured as a PeerDAS supernode
+    pub fn is_supernode(&self) -> bool {
+        self.custody_group_count == self.spec.number_of_columns
+    }
+
+    /// Returns the count of custody columns this node must sample for block import
+    pub fn custody_columns_count(&self) -> u64 {
+        self.custody_group_count
```

Comment:
This is `custody_group_count` not `custody_columns`. If `custody_columns` is what we want then we should do the conversion when setting this.

Reply:
Fixed with fea34b4b31c24f4f633b137b039615c8d7522c29